src/cow-react/modules/limitOrders/updaters/MarketPriceUpdater/hooks/useFetchMarketPrice/useHandleResponse.ts

```diff
@@ -1,17 +1,16 @@
 import { useSetAtom } from 'jotai'
 import { useCallback } from 'react'
 import { useUpdateAtom } from 'jotai/utils'
-import { Fraction } from '@uniswap/sdk-core'
-import JSBI from 'jsbi'
+import { CurrencyAmount, Percent, Price } from '@uniswap/sdk-core'
 
 import { SimpleGetQuoteResponse } from '@cowprotocol/cow-sdk'
 import { useLimitOrdersTradeState } from '@cow/modules/limitOrders/hooks/useLimitOrdersTradeState'
 import { updateLimitRateAtom } from '@cow/modules/limitOrders/state/limitRateAtom'
 import { limitOrdersQuoteAtom } from '@cow/modules/limitOrders/state/limitOrdersQuoteAtom'
-import { getDecimals } from '@cow/modules/limitOrders/utils/getDecimals'
 import { CancelableResult } from 'utils/async'
-import { MARKET_RATE_ADJUSTMENT } from '@cow/modules/limitOrders/const/trade'
-import { ONE_FRACTION } from '@src/constants/misc'
+import { FractionUtils } from '@cow/utils/fractionUtils'
+
+const PRICE_SLIPPAGE = new Percent(1, 10) // 0.1%
```

Comment:
Are you sure this is 0.1%? I believe this is 10%!
0.1% would be 1/1000

See https://github.com/cowprotocol/cowswap/blob/d0638beca73b1d961c3009ac71979bebe928c80e/src/custom/state/orders/consts.ts#L16

Reply:
Added [tests](https://github.com/cowprotocol/cowswap/pull/2093/files#diff-09a4bdaa0cd0ce58762bda4899d10f8323093ddac6b727fd145d6ed0b28c04f0) for handleLimitOrderQuoteResponse and it's working as planned :)